src/lib/Scenes/Favorites/__stories__/Favorites.story.tsx

```diff
@@ -13,7 +13,7 @@ import ArtworksRenderer from "../Components/Artworks/Relay/ArtworksRenderer"
 import CategoriesRenderer from "../Components/Categories/Relay/CategoriesRenderer"
 
 storiesOf("Favourites/Relay")
-  .add("Root", () => <Favourites />)
+  .add("Root", () => <Favourites tracking={trackingData => console.log(trackingData)} />)
```

Comment:
logs?

Reply:
ah, storybook

Reply:
Aye, this is in a story so you can see that the callback actually works